editor/components/block-list/index.js

```diff
@@ -37,6 +37,14 @@ import {
 } from '../../selectors';
 import { startMultiSelect, stopMultiSelect, multiSelect, selectBlock } from '../../actions';
 
+function isInputField( { nodeName, contentEditable } ) {
```

Comment:
Minor, but by making this generic to any Node it probably belongs somewhere where it can be found and reused. Not a blocker.

Reply:
@mcsf Okay, I moved it to utils/dom :)